src/Activity/TrainWithExercises/ShowExercise.js

```diff
@@ -30,15 +30,18 @@ class ShowExercise extends React.Component {
       studentAnswer: false
     };
   }
-
+  handleSudentAnswer = ()=>{
+    if (this.props.activityChosen === "mammals"){
+      return this.props.student.answerMammal(this.props.data.shapeFeatures);
+    }
+      return this.props.student.answerParallelogram(this.props.data.shapeFeatures);
+  }
```

Comment:
I prefer that you change `answerParallelogram` into `answer`. Later we will work on making VirtualStudents generic to several topic. For now work only with the default virtual model that gives random answers